### PR TITLE
Add Chinese Translation

### PR DESCRIPTION
**Pull request by ashishkashyap31**
Added Chinese translation.

### CODE REVIEW

**Comment by ashishkashyap31** [comment]
Solves #225  @tonybaloney .

File: package.nls.zh.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "vscodePets.gray": "灰色的",
+  "vscodePets.red": "红色的",
+  "vscodePets.white": "白色的",
+  "vscodePets.cat": "猫",
+  "vscodePets.clippy": "笨拙",
+  "vscodePets.cockatiel": "小鹦鹉",
+  "vscodePets.crab": "螃蟹",
+  "vscodePets.dog": "狗",
+  "vscodePets.rocky": "岩石",
+  "vscodePets.rubber-duck": "橡皮鸭",
```
Comment (yihong0618) [inline package.nls.zh.json:25]: 小黄鸭好一些？
little yellow duck maybe better? since we call it '小黄鸭调试'？

File: package.nls.zh.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "vscodePets.spawnPet": "产卵宠物",
```
Comment (yihong0618) [inline package.nls.zh.json:2]: 产卵宠物不太好懂吧？
‘A bit hard to understand’ maybe a bit hard to understand for Chinese?

File: package.nls.zh.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "vscodePets.spawnPet": "产卵宠物",
+  "vscodePets.petPanel": "宠物面板",
+  "vscodePets.selectPet": "选择宠物",
+  "vscodePets.selectColor": "选择颜色",
+  "vscodePets.selectPetToRemove": "选择宠物去除。",
+  "vscodePets.leaveBlank": "留空白名称",
+  "vscodePets.nameYourPet": "命名您的宠物",
+  "vscodePets.cancelledSpawningPet": "取消产卵宠物",
+  "vscodePets.spawnPetWithClosedPlayground": "已经创建了一个宠物游乐场。 现在，您可以使用\"Spawn Additional Pet\"命令添加更多宠物。",
```
Comment (BlackHole1) [inline package.nls.zh.json:10]:
```suggestion
  "vscodePets.spawnPetWithClosedPlayground": "已经创建了一个宠物游乐场。 现在，您可以使用 \"Spawn Additional Pet\" 命令添加更多宠物。",

```

File: package.nls.zh.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  "vscodePets.white": "白色的",
+  "vscodePets.cat": "猫",
+  "vscodePets.clippy": "笨拙",
+  "vscodePets.cockatiel": "小鹦鹉",
+  "vscodePets.crab": "螃蟹",
+  "vscodePets.dog": "狗",
+  "vscodePets.rocky": "岩石",
+  "vscodePets.rubber-duck": "橡皮鸭",
+  "vscodePets.snake": "蛇",
+  "vscodePets.totoro": "托托罗",
```
Comment (BlackHole1) [inline package.nls.zh.json:27]:
```suggestion
  "vscodePets.totoro": "龙猫",

```

File: package.nls.zh.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  "vscodePets.black": "黑色的",
+  "vscodePets.brown": "棕色的",
+  "vscodePets.green": "绿色的",
+  "vscodePets.yellow": "黄色",
+  "vscodePets.gray": "灰色的",
+  "vscodePets.red": "红色的",
+  "vscodePets.white": "白色的",
+  "vscodePets.cat": "猫",
+  "vscodePets.clippy": "笨拙",
+  "vscodePets.cockatiel": "小鹦鹉",
```
Comment (BlackHole1) [inline package.nls.zh.json:21]:
```suggestion
  "vscodePets.cockatiel": "鹦鹉",

```

File: package.nls.zh.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  "vscodePets.crab": "螃蟹",
+  "vscodePets.dog": "狗",
+  "vscodePets.rocky": "岩石",
+  "vscodePets.rubber-duck": "橡皮鸭",
+  "vscodePets.snake": "蛇",
+  "vscodePets.totoro": "托托罗",
+  "vscodePets.zappy": "活泼的"
```
Comment (BlackHole1) [inline package.nls.zh.json:28]:
```suggestion
  "vscodePets.zappy": "zappy"

```

It is a name that we should not translate

File: package.nls.zh.json (hunk 2 above)
Comment (BlackHole1) [inline package.nls.zh.json:2]:
```suggestion
  "vscodePets.spawnPet": "创建宠物",

```

File: package.nls.zh.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
+{
+  "vscodePets.spawnPet": "产卵宠物",
+  "vscodePets.petPanel": "宠物面板",
+  "vscodePets.selectPet": "选择宠物",
+  "vscodePets.selectColor": "选择颜色",
+  "vscodePets.selectPetToRemove": "选择宠物去除。",
+  "vscodePets.leaveBlank": "留空白名称",
+  "vscodePets.nameYourPet": "命名您的宠物",
+  "vscodePets.cancelledSpawningPet": "取消产卵宠物",
```
Comment (BlackHole1) [inline package.nls.zh.json:9]:
```suggestion
  "vscodePets.cancelledSpawningPet": "取消创建宠物",

```

File: package.nls.zh.json [hunk 8]
```diff
@@ -0,0 +1,29 @@
+{
+  "vscodePets.spawnPet": "产卵宠物",
+  "vscodePets.petPanel": "宠物面板",
+  "vscodePets.selectPet": "选择宠物",
+  "vscodePets.selectColor": "选择颜色",
+  "vscodePets.selectPetToRemove": "选择宠物去除。",
+  "vscodePets.leaveBlank": "留空白名称",
```
Comment (BlackHole1) [inline package.nls.zh.json:7]:
```suggestion
  "vscodePets.leaveBlank": "留空则随机一个名称",

```

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

File: package.nls.zh.json (hunk 4 above)
Comment (yihong0618) [reply package.nls.zh.json:27]: cool

**Comment by ashishkashyap31** [comment]
@BlackHole1 @yihong0618 , did the changes as requested, please review.

**Review by yihong0618 — APPROVED** [review]
LGTM nice job~

**Review by tonybaloney — APPROVED** [review]

**Comment by tonybaloney** [comment]
Thanks everyone.

**Pull request merged**